src/message_pack/any.cr

```diff
@@ -0,0 +1,135 @@
+module MessagePack
+  # A wrapper around a MessagePack object with convenience methods for type conversions
+  struct Any
+    getter raw : Type
+
+    def initialize(@raw)
+    end
+
+    def self.new(unpacker : IOUnpacker)
+      new(unpacker.read)
+    end
+
+    def self.from_msgpack(io)
+      new(MessagePack.unpack(io))
+    end
+
+    def dup
+      Any.new @raw.dup
+    end
+
+    def clone
+      Any.new @raw.clone
+    end
+
+    {% for name, type in {
+                           nil:    Nil,
+                           bool:   Bool,
+                           float:  Float64,
+                           string: String,
+                           hash:   Hash(Type, Type),
+                           array:  Array(Type),
+                           bytes:  Bytes,
```

Comment:
since Bytes not merged, would it work?

Reply:
Removed, it worked because nothing was calling it